main/autocode/src/org/ejml/GenerateJavaCode32.java

```diff
@@ -60,6 +60,14 @@ public GenerateJavaCode32() {
         prefix32.add("FMonoids");
         prefix64.add("DSemiRings");
         prefix32.add("FSemiRings");
+        prefix64.add("DMasks");
```

Comment:
What do you think about making DMask the prefix for all of these classes so that can reduce the number of special cases? E.g. SparseDMask -> DMaskSparse

Reply:
Looks much more consistent :)